### PR TITLE
make adjustments to latest guidelines

### PR DESCRIPTION
**Pull request by arif-ali**
Initial draft to move this snap over to the new process

### CODE REVIEW

File: .github/workflows/release-to-candidate.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    strategy:
+      matrix:
+        architecture: ${{ fromJSON(needs.get-architectures.outputs.architectures-list) }}
+    steps:
+      - name: 🚢 Release to latest/candidate
+        uses: snapcrafters/ci/release-to-candidate@main
+        with:
+          architecture: ${{ matrix.architecture }}
+          launchpad-token: ${{ secrets.LP_BUILD_SECRET }}
+          store-token: ${{ secrets.SNAP_STORE_CANDIDATE }}
```
Comment (jnsgruk) [inline .github/workflows/release-to-candidate.yaml]:
```suggestion
          launchpad-token: ${{ secrets.LP_BUILD_SECRET }}
          repo-token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
          store-token: ${{ secrets.SNAP_STORE_CANDIDATE }}
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+<h1 align="center">
+  <img src="https://avatars1.githubusercontent.com/u/29598503?v=3&s=256" alt="mutt">
+  <br />
+  mutt
+</h1>
+
+<p align="center"><b>This is the snap for mutt</b>, <i>“Mutt is a sophisticated text-based Mail User Agent.”</i>. It works on Ubuntu, Fedora, Debian, and other major Linux
+distributions.</p>
+
+<p align="center">
```
Comment (jnsgruk) [inline README.md:10]: Can you add CI badges in here (like [this](https://github.com/snapcrafters/signal-desktop/blob/d581150fdf8e46bbdb1ea8a1fdce278330a8b2ef/README.md?plain=1#L11-L13)) :slightly_smiling_face:

File: README.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+| [Your Name](https://github.com/yourname/) |
+--> 
+
+<!-- Uncomment and modify this when you have upstream contacts
+## Upstream
+
+| [![Upstream Name](https://gravatar.com/avatar/bc0bced65e963eb5c3a16cab8b004431?s=128)](https://github.com/upstreamname) |
+| :---: |
+| [Upstream Name](https://github.com/upstreamname) |
+-->
```
Comment (jnsgruk) [inline README.md]: Please update this to the same template as the Signal README if you can! :)

Comment (arif-ali) [reply README.md]: oh, missed those. I'll get that amended

**Comment by jnsgruk** [comment]
@arif-ali couple of minor changes here to get this across the line - do you need any assistance at all? :slightly_smiling_face:

**Comment by arif-ali** [comment]
> @arif-ali couple of minor changes here to get this across the line - do you need any assistance at all? 🙂

Just time, have it ready, just need to push. Will have a look tomorrow

**Review by jnsgruk — APPROVED** [review]

**Pull request merged**